src/misc/cache/lv_image_cache.c

```diff
@@ -49,14 +47,14 @@ static void image_cache_free_cb(lv_image_cache_data_t * entry, void * user_data)
  *   GLOBAL FUNCTIONS
  **********************/
 
-lv_result_t lv_image_cache_init(void)
+lv_result_t lv_image_cache_init(uint32_t size)
 {
     if(img_cache_p != NULL) {
```

Comment:
Should we check invalid size para too?
```suggestion
    if(img_cache_p != NULL || size == 0) {
```

Reply:
zero is ok, it accepts zero